packages/sdk/contractkit/src/wrappers/Election.ts

```diff
@@ -355,14 +355,26 @@ export class ElectionWrapper extends BaseWrapperForGoverning<Election> {
     )
   }
 
+  /**
+   * Creates a transaction object for revoking active votes.
+   * @dev In cases where `revokeActive` transaction object will be executed right after
+   * a `revokePending` transaction object, one must use the total revoke amount to account
+   * for any removed pending votes when obtaining the `findLesserAndGreaterAfterVote`.
+   * @param value Amount to be removed from active votes.
+   * @param totalValue Amount used to obtain the lesser and greater addresses.
```

Comment:
as someone who isn't familiar with this Im not sure what I would put into totalValue. total of what?